lib/modulesync.rb

```diff
@@ -124,10 +123,10 @@ def self.manage_module(puppet_module, module_files, module_options, defaults, op
     files_to_manage.each { |filename| manage_file(filename, settings, options) }
 
     if options[:noop]
-      Git.update_noop(module_name, options)
+      Git.update_noop(git_repo, options)
     elsif !options[:offline]
       # Git.update() returns a boolean: true if files were pushed, false if not.
-      pushed = Git.update(module_name, files_to_manage, options)
+      pushed = Git.update(git_repo, files_to_manage, options)
```

Comment:
@ekohl Interestingly, even though the tests pass this may turn out to be an issue. Because in [Git.update](https://github.com/vshn/modulesync/blob/bb5c88394105d0a5696c670db985932d5a410ea3/lib/modulesync/git.rb#L118) the `git_repo` value is prepended with a `:project_root` value. Not sure, though, honestly. (Ruby is not my first language.)

Reply:
This makes sense now, chiming in with [my discovery above](https://github.com/voxpupuli/modulesync/pull/152#discussion_r201506920): The `:project_root` has nothing to do with Git. It's just the place where all Git operations are meant to be performed in.

The `Git.update` only does things in the final _Git_ project root = `:project_root` + Git repository path (even if this is two levels).